learn/advanced/snapshots_vs_dumps.md

```diff
@@ -2,32 +2,31 @@
 
 Meilisearch has two ways to backup its data: `snapshots` and `dumps`.
 
-### Snapshots
+## Snapshots
 
-**Snapshots** make it possible to schedule the creation of hard copies of your database.
+Snapshots make it possible to schedule the creation of hard copies of your database. These copies are bound to a specific Meilisearch version.
 
-This feature is **intended mainly as a safeguard**: ensuring that if some failure occurs, you're able to relaunch your database quickly and efficiently from a snapshot.
+This feature is **intended mainly as a safeguard**: if something goes wrong in an instance, you're able to relaunch your database quickly and efficiently.
 
-The documents in a snapshot are already "indexed" and ready to go, greatly increasing import speed. However, as a result, **snapshots are not compatible between different versions of Meilisearch**.
+The documents in a snapshot are already indexed and ready to go, greatly increasing import speed. However, **snapshots are not compatible between different versions of Meilisearch**.
```

Comment:
After discussing this with @irevoire and the @meilisearch/docs-team, we have decided to keep things simple. Since it is not completely impossible a patch breaks compatibility, it's safer to push dumps as the officially supported method of migration between versions.